tests/git/utils.ts

```diff
@@ -1,72 +1,270 @@
-import type { EncryptedFS } from 'encryptedfs';
-import type { ReadCommitResult } from 'isomorphic-git';
+import type { FileSystem, POJO } from '@';
+import type { CapabilityList } from '@/git/types';
+import type { Arbitrary } from 'fast-check';
+import type fs from 'fs';
 import path from 'path';
 import git from 'isomorphic-git';
+import fc from 'fast-check';
+import * as gitUtils from '@/git/utils';
+import * as gitHttp from '@/git/http';
+import * as utils from '@/utils';
 
+// Just to avoid confusing the type with the name
+type FsType = typeof fs;
```

Comment:
Can we use the filesystem type?

There's already a `import type fs from 'fs'` in `src/types.ts` in Polykey. Probably one in PK CLI too?

Reply:
It makes more sense to use `FileSystemWritable` or `EncryptedFs` as the type here. `isomorphic-git` doesn't like the `FileSystem` type for reasons I haven't dug into.

I think I'll just make sure everything that uses the `fs` works with the `EFS` and for testing we use `fs` but override the type where needed.

Reply:
Switched everything relevant to `efs: EncryptedFs`. This should make the domain more consistent.